modules/rostests/kmtests/ntos_cc/CcCopyRead_drv.c

```diff
@@ -315,17 +326,23 @@ TestIrpHandler(
         Status = STATUS_SUCCESS;
     }
 
+    // CcCopyRead never sets STATUS_PENDING, especially with wait=TRUE.
```

Comment:
Who says this status must come from CcCopyRead?

Reply:
`CcCopyRead` block is the only place this test is not explicitly setting `Status`.